roles/inherit_kerberos/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: "Install client packages on Red Hat based distributions"
+  ansible.builtin.dnf:
+    name:
+      - "krb5-workstation"
+      - "krb5-libs"
+    state: present
+  when: ansible_os_family == "RedHat"
+
+- name: "Install client packages on Debian based distributions"
+  ansible.builtin.apt:
+    name: "krb5-user"
+    state: present
+  when: ansible_os_family == "Debian"
+
+- name: "Copy Kerberos client configuration from Host"
+  ansible.builtin.copy:
+    src: "{{ inherit_kerberos_config }}"
+    dest: /etc/krb5.conf
+    owner: root
+    group: root
+    mode: '0644'
+  when: inherit_kerberos_config is defined
+
+- name: "Copy Kerberos credential cache from Host"
+  ansible.builtin.copy:
+    src: "{{ inherit_kerberos_ccache }}"
+    dest: "{{ inherit_kerberos_ccache }}"
+    owner: "{{ inherit_kerberos_local_user_name }}"
```

Comment:
where does `inherit_kerberos_local_user_name` come from? I don't see it defined anywhere.

Reply:
This was originally written to be part of the `unprivileged_user` role and the value of `unprivileged_user_username` was passed for the owner. The approach was to give `unprivileged_user` additional features to have more general user creation/setup powers, while leaving the default behavior essentially the same in setting up the `vagrant` user for images that don't already have it; therefore the assumption at that time was that this block does not run by default, so an user passing the extra stuff required to run these extra bits would be doing it to setup their personal or devel user and not the default `vagrant` user in that role.

So with the request to split this out into more of a standalone role, instead I assume that the value will be provided by whatever role or playbook includes this role.

Reply:
Ah, got it!